### PR TITLE
Style improvements

### PR DESCRIPTION
**Pull request by capsia37**
Hello!
This PR contains some of the style improvements requested:

- Fix spacing on larger displays
- Fix colors for code blocks in light theme
- Fix the table of contents by moving content in separate pages
- Improve dark mode compatibility ( some images have now transparent background and/or have inverted colors in dark mode )
- Replaced an image containing code with a code block

### CODE REVIEW

File: appdev/code-editor.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Code Editor Integrations
```
Comment (doniks) [inline appdev/code-editor.rst:1]: ?

File: appdev/external-docs.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Documentation
```
Comment (doniks) [inline appdev/external-docs.rst:1]: ?

File: appdev/guides/importing-CH-urldispatcher.rst [hunk 3]
```diff
@@ -1,8 +1,16 @@
 Importing from Content Hub and URLdispatcher
 ============================================
 
-.. figure:: /_static/images/appdev/guides/importingCHdispatcherimages/01ichu.png
-        :align: center
+.. code:: javascript
+
+        "hooks": {
```
Comment (doniks) [inline appdev/guides/importing-CH-urldispatcher.rst:6]: ?

File: appdev/index.rst [hunk 4]
```diff
@@ -3,119 +3,17 @@ App development
 
 Great!, you'd like to develop an app for Ubuntu Touch. Here are some resources that we hope help you get started. 
 
-If you have questions that aren't answered in the docs or want to chat with other Ubuntu Touch developers? Join our `app dev Telegram group <https://t.me/UbuntuAppDevEN>`_ or chat with us on the `UBports app dev forum <https://forums.ubports.com/category/35/app-development>`_.
```
Comment (doniks) [inline appdev/index.rst:6]: ?

**Review by doniks — CHANGES_REQUESTED** [review]
Hey @capsia37 I see a couple of content changes. But you don't mention it in your commits. Are those intentional? I mark a few examples with ? below, but there are more

**Comment by capsia37** [comment]
Hello! Yes, it is intentional, the content is the same as before, it just got distributed in pages because of the way that the table of contents works in the theme. When we changed the theme we created some ambiguity in the structure of the table of contents since the headings of the pages are displayed in the right sidebar to reduce the depth of the left sidebar which is now cleaner. But this also caused the left sidebar to group all the pages together instead of grouping them by chapter ( which are listed on the right sidebar ). This changes should bring the structure of the left menu back to what it used to be.

Previous toc:
Previous

Current toc:
Current

Expected toc:
Expected

**Comment by doniks** [comment]
Ah, yes, now I understand. Great, I think independently of adjusting for the theme, this actually makes the docs code a little less spaghetti. Nice!

And wow really nice dark theme support. I love it inverting images :O I didn't know that's possible. Thanks a lot!

**Review by doniks — APPROVED** [review]

**Comment by doniks** [comment]
@capsia37 I feel we should get your recent style work get highlighted in the next Q&A. 

Firstly, just the revamped 'normal' look in UBports style is worth noting. But also, secondly the dark mode look could be celebrated . However, for the dark mode there is one sore point: the logo on the top left. This doesn't look good. It's very dull. Are you able to adjust this?

**Comment by doniks** [comment]
image

**Pull request merged**